### PR TITLE
Changes After First Review

### PR DESCRIPTION
**Pull request by kubilaycaglayan**
[Click For Live Website](https://rawcdn.githack.com/kblycaglayan/youtube-page/first-review/index.html)

**Navbar:** We set different divisions for the elements and put them in order.
**Main Video Section:** We improved the title placement and vision of the page.
**Recommended Video Section:** We put the titles on the side with the view counts and resized the thumbnails.
**Descriptive Readme:** We prepared a descriptive readme file for our project based on our TSE's past projects.
**Live Link:** We created a link for the easiness of the job.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,75 +1,187 @@
...
+                src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/1f/Echo_%28Notifications%29_logo.svg/768px-Echo_%28Notifications%29_logo.svg.png" alt="Sign in">
+            <img src="images/Untitled.png" alt="Notifications">
+        </div>
+
+    </div>
+
+    <div class="main-section">
+        <div id="video-section">
+            <div id="video">
+                <iframe width="560" height="315" src="https://www.youtube.com/embed/VtM9Doxw878"
```
Comment (euqueme) [inline index.html]: remove inline styling the width and height selectors should be put in the CSS file

File: index.html [hunk 2]
```diff
@@ -1,75 +1,187 @@
...
+                    allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                    allowfullscreen></iframe>
+            </div>
+            <p>The Genuine Origin Coffee Project | Brazil</p>
+
+            <div>
+                <img src="images/murilo.png"
+                        alt="Avatar" height="50" width="50">
```
Comment (euqueme) [inline index.html]: remove inline styling the width and height selectors should be put in the CSS file

File: index.html [hunk 3]
```diff
@@ -1,75 +1,187 @@
...
+                <div id="view-count" class="video-bottom">
+                    <p>View count: 15,600</p>
+                </div>
+                <div id="like-buttons" class="video-bottom">
+                    <img src="images/share.png"
+                        alt="Share">
+                    <img src="images/thumbs.png"
+                        alt="Like and dislike buttons.">
+                    <img src="https://i.pinimg.com/originals/91/e9/65/91e965a8c2f1556057fe722e0c023949.jpg"
+                        alt="Subscribe" width="100" height="100">
```
Comment (euqueme) [inline index.html]: remove inline styling the width and height selectors should be put in the CSS file

File: index.html [hunk 4]
```diff
@@ -1,75 +1,187 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (euqueme) [inline index.html]: remove empty lines from #95 to #111

File: index.html [hunk 5]
```diff
@@ -1,75 +1,187 @@
...
+<li><img src="https://www.brandeps.com/icon-download/C/Cloud-upload-icon-vector-06.svg"></li>
+<li><img id="notification-img"
+src="https://upload.wikimedia.org/wikipedia/commons/thumb/1/1f/Echo_%28Notifications%29_logo.svg/768px-Echo_%28Notifications%29_logo.svg.png">
+</li>
+<li><img src="images/Untitled.png"></li>
+</nav>
+<ul>
+<li class="body">
+<h1>This our Brazillian Coffee!</h1> 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/VtM9Doxw878" frameborder="0"
```
Comment (euqueme) [inline index.html]: remove inline styling the width and height selectors should be put in the CSS file

File: index.html [hunk 6]
```diff
@@ -1,75 +1,187 @@
...
+</li>
+
+<li><img src="https://image.freepik.com/free-vector/like-dislike-flat-icons_87538-584.jpg"
+alt="Like and dislike buttons." width="100" height="100"></li>
+<li><img src="https://thumbs.dreamstime.com/z/share-button-rounded-isolated-sign-158485440.jpg"
+alt="Share Button" width="100" height="100"></li>
+<li><img src="https://cdn4.iconfinder.com/data/icons/user-avatar-flat-icons/512/User_Avatar-04-512.png"
+alt="Avatar" width="100" height="100"></li>
+<li><img src="https://i.pinimg.com/originals/91/e9/65/91e965a8c2f1556057fe722e0c023949.jpg"
+alt="Subscribe" width="100" height="100"></li>
```
Comment (euqueme) [inline index.html]: remove inline styling the width and height selectors should be put in the CSS file

File: styles/style.css [hunk 7]
```diff
@@ -50,4 +131,18 @@ border: solid black;
...
+
+#notification-img {
+    
+    width: 50px;
+
+}
+
+#views {
+    font-size: 20;
+} */
```
Comment (euqueme) [inline styles/style.css]: remove commented code from line #79 to line #148

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @kblycaglayan and @MuriloRoque  :wave: 

###  Thank you for doing the changes requested by the previous reviewer :medal_sports: I see you did the changes in the previous pull request, do not open a new pull request just do the following changes, and submit for a code review using the same pull request #10 

## Page structure

**Video Size**

- [x] The video section should be bigger please fix it

- Yours

image

- Original

image

## Generals

**Inline styling**

- [ ] please remove all inline styling from your HTML, all CSS selectors should be declared in the CSS file, I have commented your code, where I've found inline styling :arrow_double_down: 

**Code commenting**

- [x] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history, I have also add comments where commented code should be deleted :arrow_double_down: 

**HTML validations**

Please fix the errors found by [validator](https://validator.w3.org/)

image

## That's it! :raised_hands: 

## Please :pray:  do the changes :memo:  and submit back for a code review when you are done :rocket: 

Happy coding! :woman_technologist: 

Any questions? reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Comment by kubilaycaglayan** [comment]
[**Click For Live Website**](https://raw.githack.com/kblycaglayan/youtube-page/first-review/index.html)

**Main Video Section**: We made it bigger by adjusting the position percentages.
**Clean Code**: We deleted the unnecessary comment lines.
**Styling:** We removed the inline styling from HTML file.
**Stickler:** We added stickler to check our linter errors.

Thanks for your time and efforts...

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello Team!

### Good job so far, I see you have made the previously required changes by my colleagues. However you still have a few changes you need to work on.
- Rebuilt the user details section to look like the one in youtube
details
- We expect something similar to this
expected
- Validate your html code on this [online tool](https://validator.w3.org/#validate_by_upload) and resolve the errors
validate
- Add the comments section for your featured video.
- On your sidebar, work on the spacing make look like youtube one
spacing

### Make these changes and resubmit for a new review
- For assistance, contact me on @rOluochKe
- Happy coding!

**Comment by MuriloRoque** [comment]
**[Click For Live Website](https://raw.githack.com/kblycaglayan/youtube-page/first-review/index.html)**

**User Details Section:** We rebuilt the user details section to match the original website.
**Recommended videos section:** We adjusted the size of the images and fonts to match the original website.
**Validation:** We used the online tool you recommended us to check for errors.

Thanks for your time and efforts...

**Comment by davidmrtz-dev** [comment]
### **Status: Changes required.** 🛑 👀 🔍 
### Hi Team. 😄 
### Nice job so far but we have some issues to solve before move on. 💯 👍 

- [ ] **Let's check the comments section:**

**Here's how it looks in your project:**

Screen Shot 2020-02-20 at 7 30 40 AM
**There's no comment section**

**Here's how it suppose to looks:** 
Screen Shot 2020-02-20 at 7 28 50 AM

- [ ] **There are some stickler CSS issues, please fix that. 🙏 🔍** 
- [ ] **Let's check the HTML on the validator, we have one warning:**

Screen Shot 2020-02-20 at 7 33 16 AM


Please make the changes required and submit a new code review once you are ready! 🙏 

Happy coding! 😄 💻 

**[TSE:  David Elí]** 👨‍💻 
Github: [@davitomix](https://github.com/davitomix)
Twitter: [@davitomix](https://twitter.com/davitomix007)
Linkedin: [David Elí Martínez García](https://www.linkedin.com/in/davideligarcia/)

**Comment by kubilaycaglayan** [comment]
[**Click For Live Website**](https://raw.githack.com/kblycaglayan/youtube-page/first-review/index.html)

**CSS issues**: Fixed.
**HTML on the validator**: Fixed
**Comments Section**: Added.

**Comment by lekegitrepo** [comment]
This PR is under Review by: **_[TSE] Temitayo Adeleke_**

**Comment by lekegitrepo** [comment]
## Well Done Team!!! :100: 
This project is **_Approved_** you can merge to master now.
Good luck with your next project.

### Reviewed by: _Temitayo Adeleke_

**Review by lekegitrepo — APPROVED** [review]
# Approved

**Pull request merged**